doc/package-notes.xml

```diff
@@ -654,6 +654,68 @@ overrides = self: super: rec {
 </screen>
   </section>
  </section>
+ <section xml:id="sec-firefox">
+  <title>Firefox</title>
+
+  <section xml:id="sec-firefox-config">
+   <title>Configuring the Firefox Wrapper</title>
+
+   <para>
+    The firefox (wrapper) package can be configured through several mechanisms.
+    The older
```

Comment:
I don't think the old/new wording fits here. It suggests that the "new" method is a replacement for the old one, which it is not.